next.config.js

```diff
@@ -4,9 +4,12 @@ const withCSS = require('@zeit/next-css');
 const withImages = require('next-images');
 const webpack = require('webpack');
 const lessToJS = require('less-vars-to-js');
+
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const TerserPlugin = require('terser-webpack-plugin');
 const OptimizeCssPlugin = require('optimize-css-assets-webpack-plugin');
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
+
```

Comment:
Reducing bundle size by not including unneeded locales with Moment (~ 100 kB).

Reply:
fyi: I suggest that we remove `moment.js` (although I go the impression from the diagrams you posted that some other dependency is dragging it) https://biomage.atlassian.net/browse/BIOMAGE-630

Reply:
I think it's a good idea to remove it anyways. I'll do it just now.